tasks-container-update

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+# Update .cockpit-ci/container to the latest tasks container tag automatically
+
+import argparse
+import asyncio
+import logging
+import sys
+from pathlib import Path
+
+import aiohttp
+from yarl import URL
+
+import task
+from lib.aio.jsonutil import get_str, get_strv, typechecked
+
+sys.dont_write_bytecode = True
+
+
+async def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--debug', action='store_true')
+    parser.add_argument('--dry-run', '-n', action='store_true')
+    parser.add_argument('--image', default='ghcr.io/cockpit-project/tasks')
+    parser.add_argument('--file', default='.cockpit-ci/container')
+    args = parser.parse_args()
+
+    if args.debug:
+        logging.basicConfig(level=logging.DEBUG)
+
+    headers = {
+        'User-Agent': 'cockpit-project/bots (tasks-container-update)'
+    }
+
+    service, _, repository = args.image.partition('/')
+    if not service or not repository:
+        args.error(f'Invalid image name: {args.image}')
+
+    async with aiohttp.ClientSession(raise_for_status=True) as session:
+        async with session.get(URL.build(scheme='https', host=service, path='/token', query={
```

Comment:
This could use a comment where this API came from (e.g. you got it from `skopeo --debug`, right?)

Reply:
I added a note to the commit message (as well as links to the official documentation).